library/network/src/lib/y2firewall/firewalld/service.rb

```diff
@@ -0,0 +1,92 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2017 SUSE LLC
+#
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact SUSE.
+#
+# To contact SUSE about this file by physical or electronic mail, you may find
+# current contact information at www.suse.com.
+# ------------------------------------------------------------------------------
+
+require "yast"
+require "y2firewall/firewalld/api"
+require "y2firewall/firewalld/relations"
+
+module Y2Firewall
+  class Firewalld
+    # Class to work with Firewalld services
+    class Service
+      extend Relations
+      include Yast::I18n
+      extend Yast::I18n
+
+      attr_accessor :name
+      attr_accessor :short
+      attr_accessor :description
```

Comment:
just readers should be enough ...

Reply:
If we are not going to allow to define them then sure, just read is enough